vars/piperPipelineStageArtifactDeployment.groovy

```diff
@@ -0,0 +1,118 @@
+import com.sap.piper.ConfigurationHelper
+import com.sap.piper.DownloadCacheUtils
+import com.sap.piper.GenerateStageDocumentation
+import com.sap.piper.ReportAggregator
+import com.sap.piper.Utils
+import groovy.transform.Field
+
+import static groovy.json.JsonOutput.toJson
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field String STEP_NAME = getClass().getName()
+
+@Field Set GENERAL_CONFIG_KEYS = []
+@Field Set STAGE_STEP_KEYS = [
```

Comment:
I would like to get rid of a configuration definition on stage level. I think the step could read this information from the config.